pkg/syncer/cnsoperator/controller/cnsnodevmattachment/cnsnodevmattachment_controller.go

```diff
@@ -541,6 +569,38 @@ func removeFinalizerFromCRDInstance(ctx context.Context,
 	}
 }
 
+// addFinalizerToPVC will add the CNS Finalizer, cns.vmware.com/pvc-protection,
+// from a given PersistentVolumeClaim.
+func addFinalizerToPVC(ctx context.Context, client client.Client, pvc *v1.PersistentVolumeClaim) {
+	log := logger.GetLogger(ctx)
+	pvc.Finalizers = append(pvc.Finalizers, cnsoperatortypes.CNSPvcFinalizer)
+	log.Debugf("Removing %q finalizer from PersistentVolumeClaim: %q on namespace: %q",
+		cnsoperatortypes.CNSPvcFinalizer, pvc.Name, pvc.Namespace)
+	err := client.Update(ctx, pvc)
+	if err != nil {
```

Comment:
Why are we ignoring the error? Shouldn't the next reconcile of CnsNodeVmAttachment attempt to add the finalizer?

Reply:
Right, look like a miss. Updated the error handling path